Manage/DeployWorkspace/azuredeploy.json

```diff
@@ -2,23 +2,29 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "name": {
+        "workspaceName": {
             "type": "String",
             "metadata": {
-                "description": "The name of the workspace."
+                "description": "The name of the Workspace."
             }
         },
-        "storage": {
+        "storageAccountName": {
             "type": "String",
             "metadata": {
-                "description": "The name of the storage. If not provided, the workspace name will be used."
+                "description": "The name of the primary ADLS Gen2 Storage Account. If not provided, the workspace name will be used."
             }
         },
         "sqlAdministratorLogin": {
-            "type": "String"
+            "type": "String",
+            "metadata": {
+                "description": "Provide the user name for SQL server login."
+            }
         },
         "sqlAdministratorPassword": {
-            "type": "SecureString"
+            "type": "SecureString",
+            "metadata": {
+                "description": "The password for the SQL server. Visit https://aka.ms/azuresqlserverpasswordpolicy for password policy."
```

Comment:
Change it to the password for the serverless SQL pool endpoint. there is no SQL Server in Synapse.

Reply:
Updated this.